environment.yml

```diff
@@ -0,0 +1,10 @@
+name: csmswig
+channels:
+  - conda-forge
+  - usgs-astrogeology
+dependencies:
+  - cmake >= 3.10
+  - libcsm
+  - python
+  - swig
+  - numpy
```

Comment:
All py dependencies in one place now, not scattered in the .travis.yml

Reply:
:+1: